.ci/pipeline/build-and-test-lnx.yml

```diff
@@ -15,58 +15,58 @@
 #===============================================================================
 steps:
   - script: sudo apt-get update && sudo apt-get install -y clang-format
-    displayName: 'apt-get'
+    displayName: "apt-get"
   - script: |
       bash .ci/scripts/install_dpcpp.sh
-    displayName: 'dpcpp installation'
+    displayName: "dpcpp installation"
   - script: |
       bash .ci/scripts/describe_system.sh
-    displayName: 'System info'
+    displayName: "System info"
   - script: |
-      conda update -y -q conda
-      if [ $(echo $(PYTHON_VERSION) | grep '3.8\|3.9\|3.10') ]; then export DPCPP_PACKAGE="dpctl>=0.14 "; else export DPCPP_PACKAGE=""; fi
+      conda update -y -q conda && \
+      if [ $(echo $(PYTHON_VERSION) | grep '3.8\|3.9\|3.10') ]; then export DPCPP_PACKAGE="dpctl>=0.14 "; else export DPCPP_PACKAGE=""; fi && \
       conda create -q -y -n CB -c conda-forge -c intel python=$(PYTHON_VERSION) dal-devel mpich pyyaml $DPCPP_PACKAGE dpcpp-cpp-rt>=2023.2.0
-    displayName: 'Conda create'
+    displayName: "Conda create"
   - script: |
-      . /usr/share/miniconda/etc/profile.d/conda.sh
-      conda activate CB
-      pip install -r dependencies-dev
+      . /usr/share/miniconda/etc/profile.d/conda.sh && \
+      conda activate CB && \
+      pip install -r dependencies-dev && \
       pip list
-    displayName: 'Install develop requirements'
+    displayName: "Install develop requirements"
   - script: |
-      export DPCPPROOT=/opt/intel/oneapi/compiler/latest
-      . /usr/share/miniconda/etc/profile.d/conda.sh
-      conda activate CB
-      export DALROOT=$CONDA_PREFIX
-      ./conda-recipe/build.sh
+      export DPCPPROOT=/opt/intel/oneapi/compiler/latest && \
+      . /usr/share/miniconda/etc/profile.d/conda.sh && \
+      conda activate CB && \
+      export DALROOT=$CONDA_PREFIX && \
+      ./conda-recipe/build.sh && \
       python setup_sklearnex.py install --single-version-externally-managed --record=record_sklearnex.txt
-    displayName: 'Build daal4py/sklearnex'
+    displayName: "Build daal4py/sklearnex"
   - script: |
-      . /usr/share/miniconda/etc/profile.d/conda.sh
-      conda activate CB
-      bash .ci/scripts/setup_sklearn.sh $(SKLEARN_VERSION)
-      pip install --upgrade -r requirements-test.txt -r requirements-test-optional.txt
-      pip install $(python .ci/scripts/get_compatible_scipy_version.py)
-      if [ $(echo $(PYTHON_VERSION) | grep '3.8\|3.9\|3.10') ]; then conda install -q -y -c intel dpnp; fi
+      . /usr/share/miniconda/etc/profile.d/conda.sh && \
+      conda activate CB && \
+      bash .ci/scripts/setup_sklearn.sh $(SKLEARN_VERSION) && \
+      pip install --upgrade -r requirements-test.txt -r requirements-test-optional.txt && \
+      pip install $(python .ci/scripts/get_compatible_scipy_version.py) && \
+      if [ $(echo $(PYTHON_VERSION) | grep '3.8\|3.9\|3.10') ]; then conda install -q -y -c intel dpnp; fi && \

```

Comment:
Is there a smarter way to immediately exit script on fail like using of shell init script to set it?

Reply:
Didn't find anything in the azure docs about it :( In any case, I'm moving this into a separate PR
https://github.com/intel/scikit-learn-intelex/pull/1552

Reply:
More precisely. I think `set -e` could work (didn't test it). But using `&& \` is pretty common in Dockerfiles or similar script files. That's why I chose this syntax. It works and it is explicit. I like it.